### PR TITLE
notification show text done on client part

### PR DESCRIPTION
**Pull request by Vova-Romaniuk**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/users/maryanchuk04/projects/3)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: StudCity.Frontend/src/UI/ToggleButton/index.jsx [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 	return (
-		<label className="relative inline-flex items-center my-auto cursor-pointer">
-			<input type="checkbox" value="" className="sr-only peer" />
-			<div className="w-11 h-6 bg-primaryAuthentication/70 dark:peer-focus:ring-transparent rounded-full peer dark:bg-customGray-700 peer-checked:after:translate-x-full after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-customGray-600 peer-checked:bg-green-700 after:outline-0 after:border-0 outline-0 border-0 "></div>
+		<label
+			className="relative inline-flex items-center my-auto cursor-pointer"
+		>
+			<input
+				type="checkbox"
+				value=""
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/ToggleButton/index.jsx:10]: Should `value ` be an empty string?

File: StudCity.Frontend/src/app/features/userSlice.js [hunk 2]
```diff
@@ -106,7 +125,11 @@ export const saveCurrentUser = createAsyncThunk(
 const userSlice = createSlice({
 	name: 'user',
 	initialState: initialState,
-	reducers: {},
+	reducers: {
+		toggleTextMessage: (state) => {
+			state.data.settings.textMessage = !state.data.settings.textMessage;
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/app/features/userSlice.js:130]: For now, that's great, but we get this field form API in the future. And update will be something another

File: StudCity.Frontend/src/app/features/userSlice.js [hunk 3]
```diff
@@ -162,10 +185,16 @@ export const selectUserForHeader = (state) => {
...
 
+export const { toggleTextMessage } = userSlice.actions;
+
 export const selectDataUsersFound = (state) => state.user.users;
 
 export const selectCurrentUserId = (state) => state.user.data.id;
 
 export const selectUserLanguage = (state) => state.user.data.settings.language;
 
+export const selectUserTextMessage = (state) => {
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/app/features/userSlice.js:196]: lets use arrow function

**Review by maryanchuk04 — APPROVED** [review]
Great job<3

**Review by Serhiy-dotcom — APPROVED** [review]
Nice!

**Pull request merged**